_pages/getting-started/getting-started.md

```diff
@@ -90,7 +90,7 @@ When you make the changes you want to see and open a pull request, someone will
 If pull requests and code reviews feel a bit foreign to you, it
 might be helpful to think of it as similart to suggesting changes in a shared Google Doc. 
```

Comment:
I'm not sure of the best process for pointing out a typo at this stage. "...of it as **similart** to..." Is adding a comment here okay?

Reply:
It's better than ok, it's perfect.

You can also, if you want to, click the little doc-looking icon in the comment box, and it will let you make suggestions right here in the code! Pretty spiffy!